app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,40 @@
+<div class="recipes-show-container">
+  <h2><%= @recipe.name %></h2>
```

Comment:
[OPTIONAL] It is a good practice to have an h1 on a page as it helps SEO among other benefits